### ISSUE TITLE
Automatically select unicode font from system if available

### ISSUE THREAD

**Issue by TheCycoONE** [CorsixTH/CorsixTH#823]
It would be a better experience for our international users if their system default font could be picked up and used as the unicode_font if it's available and a supported type. (e.g. true type). This way their preferred language option would be visible without navigating through the font selection screens.

**Comment by tobylane** [comment]
Finding the system font and its type would need another package, eg [fontconfig](https://github.com/LuaDist/fontconfig). #1790 chooses a highly available and suitable font, which may be the best we can do without a new dependency. Is fontconfig worth using?

**Comment by tobylane** [comment]
Yes. There isn't a reasonably complete official release of the Noto font. The two best options I've seen are from [KlokanTechNotoSansCJK-Regular.otf from klokantech](https://github.com/klokantech/klokantech-gl-fonts) or [GoNotoKurrent-Regular.ttf from satbyy](https://github.com/satbyy/go-noto-universal).
